frontend/src/Components/RoadDetails/ImageGallery.tsx

```diff
@@ -108,24 +132,28 @@ const ImageGallery: React.FC = () => {
           ))}
         </div>
       </div>
-      <button
-        className={`arrow-button left-arrow ${
-          scrollPosition === 0 ? 'disabled' : ''
-        }`}
-        onClick={() => handleScroll(-120)}
-        disabled={scrollPosition === 0}
-      >
-        &#9658;
-      </button>
-      <button
-        className={`arrow-button right-arrow ${
-          scrollPosition === (images.length - 1) * 124 ? 'disabled' : ''
-        }`}
-        onClick={() => handleScroll(120)}
-        disabled={scrollPosition === (images.length - 1) * 124}
-      >
-        &#9658;
-      </button>
+      {showArrows && (
+        <>
+          <button
+            className={`arrow-button left-arrow ${
+              scrollPosition === 0 ? 'disabled' : ''
+            }`}
+            onClick={() => handleScroll(-120)}
+            disabled={scrollPosition === 0}
+          >
+            &#9658;
+          </button>
+          <button
+            className={`arrow-button right-arrow ${
+              scrollPosition === (images.length - 1) * 124 ? 'disabled' : ''
+            }`}
+            onClick={() => handleScroll(120)}
+            disabled={scrollPosition === (images.length - 1) * 124}
+          >
```

Comment:
The button isn't disappearing and the slider is going too far away:
![image](https://github.com/DTU-SE-Group-D/LiRA-Viz/assets/65665540/bd29c32e-cb20-4e84-8930-462b59df871d)

Reply:
That issue is now fixed with the next branch update. There is no more large grey space on the left and right. The scroll function feels better. Button behavior has also changed. The users only see the buttons when the mouse is hovering over the .image-gallery container. The black arrow greys out when you can no longer move to a certain direction.

Reply:
Still isn't working on my end... If you need me to try some things to understand why it's not working, let me know